RecoTracker/MeasurementDet/plugins/TkGluedMeasurementDet.cc

```diff
@@ -139,7 +152,8 @@ bool TkGluedMeasurementDet::measurements( const TrajectoryStateOnSurface& stateO
 				      (theMonoDet->hasAllGoodChannels() || 
 				       testStrips(stateOnThisDet,gluedPlane,*theMonoDet)
 				       )
-				      ) /*Mono OK*/ || 
+				      ) /*Mono OK*/ 
+                                     && // was || 
```

Comment:
it will simplify the progress of this backport if the default behavior is unchanged.